MANIFEST.in

```diff
@@ -1,5 +1,5 @@
 include codebasin/schema/analysis.schema
 include codebasin/schema/compilation-database.schema
 include codebasin/schema/config.schema
-include codebasin/schema/coverage-0.1.0.schema
+include codebasin/schema/coverage-0.3.0.schema
```

Comment:
I think you forgot to add this file to your PR :)

Reply:
How embarrassing!  Fixed in 57f4ff8449d0874e57fbe30e750d73146df4f4fc.